packages/trader/src/App/Containers/Routes/routes.jsx

```diff
@@ -95,7 +97,11 @@ const Routes = ({
     }, []);
 
     React.useEffect(() => {
-        return () => onUnmountPortfolio();
+        if (is_reports_url) {
```

Comment:
since reports is not part of trader any more , when navigating from one of the reports routes to the trader, the trader component reloads and `onUnmountPortfolio` is going to be called. Adding this condition prevents this behavour.